package.json

```diff
@@ -25,15 +25,13 @@
     "ultron": "1.0.x"
   },
   "devDependencies": {
-    "ansi": "0.3.x",
```

Comment:
I've removed these 2 dependencies as I don't care about the fancy colors but I will add them back if this is not wanted.